File: Iniciante/URI 1010.go
package main
 
import (
    "fmt"
)
 
func main() {
    var cod1, cod2 int
    var valor1, n1, valor2, n2, total float64

    fmt.Scanf("%d", &cod1)
    fmt.Scanf("%f", &n1)
    fmt.Scanf("%f", &valor1)
    fmt.Scanf("%d", &cod2)
    fmt.Scanf("%f", &n2)
    fmt.Scanf("%f", &valor2)
	
    total = (n1 * valor1) + (n2 * valor2)
	
    fmt.Printf("VALOR A PAGAR: R$ %.2f\n", total)
}

File: Iniciante/URI 1010.php
<?php

    $line1 = explode(" ", trim(fgets(STDIN)));
    $cod1 = intval($line1[0]);
    $n1 = intval($line1[1]);
    $valor1 = floatval($line1[2]);

    $line2 = explode(" ", trim(fgets(STDIN)));
    $cod2 = intval($line2[0]);
    $n2 = intval($line2[1]);
    $valor2 = floatval($line2[2]);

    $total = ($n1 * $valor1) + ($n2 * $valor2);
    $total = number_format($total, 2, ".", "");
    echo "VALOR A PAGAR: R$ $total\n";
?>